src/renderer/components/+namespaces/namespace.store.ts

```diff
@@ -158,6 +158,12 @@ export class NamespaceStore extends KubeObjectStore<Namespace> {
     }
   }
 
+  @action
+  toggleSingle(namespace: string){
+    this.clearSelected();
+    this.selectNamespaces(namespace);
```

Comment:
This makes it so the user cannot deselect the last remaining selected namespace, which I guess is good? No point letting them pick no namespace?

Reply:
Well currently "all namespaces" and "no namespaces" are equivalent. So shift-clicking the only selected namespace would act like selecting all.